estimator/lwe.py

```diff
@@ -4,18 +4,23 @@
 """
 
 from functools import partial
+from sage.all import oo
+
 from .lwe_primal import primal_usvp, primal_bdd, primal_hybrid
 from .lwe_bkw import coded_bkw
-from .lwe_guess import exhaustive_search, mitm, distinguish  # noqa
+from .lwe_guess import exhaustive_search, mitm, distinguish, guess_composition # noqa
 from .lwe_dual import dual, dual_hybrid
-from .lwe_guess import guess_composition
 from .gb import arora_gb  # noqa
 from .lwe_parameters import LWEParameters as Parameters  # noqa
+from .conf import (red_cost_model as red_cost_model_default,
+                   red_shape_model as red_shape_model_default)
+from .util import batch_estimate, f_name
+from .reduction import RC
 
 
 class Estimate:
-    @classmethod
```

Comment:
Why?

Reply:
I can revert the change if you prefer. 

Too large an explanation for such a minor thing :D. My reasoning was that I don't see when this `rough` method is used from the class i.e. line 39 in the example doctest has `lwe.estimate.rough(schemes.Kyber512)` making use of the global `estimate` instance. This makes sense as the other method `__call__` is an instance method. Therefore `rough` is subtly not acting as most users would expect/interpret, it seems surprising. The `cls` first argument is not required by the implementation; therefore, an argument could be made that the method could be decorated as a staticmethod. However, again the situation that `__call__` would be an instance method with only one way to be called and `rough` could be called either from the class or an instance seems inconsistent.

Reply:
Yeah, okay, I see the lack of symmetry here. Let's do it as you suggest. Thanks